### ISSUE TITLE
[FEATURE] New double tap action

### ISSUE THREAD

**Issue by VasilisKos** [anilbeesetti/nextplayer#553]
Please add another option in the double tap available actions. "Back to 100% zoom": after pinching to zoom a video, a double tap would turn the video back to 100% zoom level, instead of trying to reach that by pinching again. That would be very convenient.

**Comment by anilbeesetti** [comment]
Press and hold the video zoom options button, then choose the currently selected zoom option again to reset the zoom.

**Comment by VasilisKos** [comment]
> Press and hold the video zoom options button, then choose the currently selected zoom option again to reset the zoom.

Yeah, I know, but the double tap would be a faster and more convenient way. If it's hard to implement it's ok. Close the issue.
